### DISCUSSION TITLE
Markdown sourcecode view

### DISCUSSION THREAD

**Discussion by krawallo666** [outline/outline#5324] [Ideas]
Hey all,

is it planned to implement a markdown sourcecode view? 

It would make copy+paste to other markdown tools much easier. Also copying tables would be possible as you can only mark one cell at the moment and not the table as a whole.

**Comment by tommoor** [comment]
There are no plans to create such a view. I'm not sure what you're referring to in regards to tables, you can select a cell, column, row, or the whole table using the controls that appear when focused

**Reply by krawallo666** [reply]
Thanks for your reply. 

The table controls you are referring to are only available in edit mode. Users without edit-Permissions can´t select tables properly.
